### ISSUE TITLE
Instant discussion QQ group

### ISSUE THREAD

**Issue by nttstar** [deepinsight/insightface#1069]
Group ID: 711302608.

For English developers: 

1. Install QQ international. You can google `qq international` to find your proper version.
2. Sign up a new account and log in.
3. Search group with id 711302608 to join our group.

Any problem can be left in this issue.

**Comment by uday60** [comment]
@nttstar Cannot join its all in Chinese.

**Comment by nttstar** [comment]
@uday60 You can speak English in that group, don't worry.

**Comment by uday60** [comment]
I cannot register because i cannot understand where to click. I tried taking screenshots and running ocr on it but we would need a step by step guide.

**Comment by hhakangull** [comment]
me too i can'nt register to international app

**Comment by nttstar** [comment]
I will register some accounts for you,  after a couple of days.

**Comment by nttstar** [comment]
As I saw from qq international homepage, it supports sign up with facebook account. Can you guys have a try?

**Comment by nttstar** [comment]
Please try this page for registration: https://ssl.zc.qq.com/v3/index-en.html

**Comment by xiashiwendao** [comment]
How about build a Wechat group?

**Comment by leviethung2103** [comment]
请问， 为什么我不能加群 "711302608". 他说 “拒绝让你加群”。 你可以帮我加群吗？ 我的 ID是： 3623224783. 谢谢你。

**Comment by nttstar** [comment]
> 请问， 为什么我不能加群 "711302608". 他说 “拒绝让你加群”。 你可以帮我加群吗？ 我的 ID是： 3623224783. 谢谢你。

再加一次 看到是小号拒绝了

**Comment by dexception** [comment]
@nttstar Can you please rewrite the TVM optimization module. The TVM team has rewritten the entire code.

**Comment by daywednes** [comment]
@nttstar let's create a wechat group to discuss. Would you like to take it? If not, I can help to arrange the group discussion there.

**Comment by nttstar** [comment]
@daywednes Wechat group is not good for public invitation.

**Comment by daywednes** [comment]
@nttstar how about discord?

**Comment by daywednes** [comment]
really quick question: I'm trying to understand your code - i guess the src/train.py or src/train_softmax.py is the code training the model - what's about the folders outside the src/ - guess they are used for different competitions and data. I would love to understand to write better docs for everyone to understand the code easily - happy to make a PR for you

**Comment by nttstar** [comment]
@daywednes Yes the repo is lack of full documentation. For training arcface, please use the code under ``recognition/`` now.

**Comment by faderani** [comment]
Hi @nttstar 
I couldn't register for the app and I have a problem with fine tuning R50 pretrained model. This issue has been opened multiple times but no solution was given. #1177 
can you help?

**Comment by mkhoshle** [comment]
@nttstar  Could you please help me with registration in the discussion group? It is all chineese and I do not understand the language. How can I join it?

**Comment by nttstar** [comment]
@mkhoshle PC or mobile app? You can email me.

**Comment by mkhoshle** [comment]
@nttstar  I am using pc. Ok I emailed you!

**Comment by JohannesTK** [comment]
Slack not active anymore: <img width="688" alt="Screenshot 2020-10-15 at 21 01 57" src="https://user-images.githubusercontent.com/3767980/96168619-adddc480-0f29-11eb-931a-b9fabe0e2e64.png">

Do you have a new channel @nttstar?

**Comment by nttstar** [comment]
Can you use WeChat?

**Comment by nttstar** [comment]
WeChat group updated.

**Comment by Flowingsun007** [comment]
您好，请问怎么加入微信群呢？方面放一下二维码吗

**Comment by wwidjaya** [comment]
Hi @nttstar, could you provide your wechat ID? Thanks

**Comment by mehul019** [comment]
I didn't get after applying cosine similarities, what is the matching threshold is used to make decisions whether this 2 embeddings are from same person or different?
